src/parse.rs

```diff
@@ -277,6 +278,12 @@ enum TableParseMode {
     Disabled,
 }
 
+pub struct BrokenLink<'a> {
+    pub range: ::std::ops::Range<usize>,
```

Comment:
nit: I would call this the 'span' of the link, range makes me think of number

Reply:
Good one, thanks.